### PR TITLE
full commander and safety logging by default

### PR DESCRIPTION
**Pull request by dagar**
For debugging if something goes wrong.

### CODE REVIEW

**Comment by dagar** [comment]
It looks like this would increase the typical nominal logging rate by somewhere around half a kilobyte per second with a corresponding (tiny) increase in cpu.

File: src/drivers/px4io/px4io.cpp [hunk 1]
```diff
@@ -1691,14 +1693,23 @@ PX4IO::io_handle_status(uint16_t status)
...
 	 */
-	safety_s safety{};
-	safety.timestamp = hrt_absolute_time();
-	safety.safety_switch_available = true;
-	safety.safety_off = (status & PX4IO_P_STATUS_FLAGS_SAFETY_OFF) ? true : false;
-	safety.override_available = _override_available;
-	safety.override_enabled = (status & PX4IO_P_STATUS_FLAGS_OVERRIDE) ? true : false;
-
-	_to_safety.publish(safety);
+	const bool safety_off = (status & PX4IO_P_STATUS_FLAGS_SAFETY_OFF) ? true : false;
```
Comment (bresch) [inline src/drivers/px4io/px4io.cpp:1696]:
```suggestion
	const bool safety_off = status & PX4IO_P_STATUS_FLAGS_SAFETY_OFF;
```

Comment (dagar) [reply src/drivers/px4io/px4io.cpp:1696]: Blind copy pasta. Thanks for actually reviewing.

Comment (dagar) [reply src/drivers/px4io/px4io.cpp]: Fixed, squashed, and rebased.

**Comment by dagar** [comment]
Here we go again.
Screenshot from 2020-07-17 10-30-52

**Comment by bkueng** [comment]
Why not adding the arming/disarming reason to vehicle_status? That would make it perfectly clear what the reason was.

**Comment by dagar** [comment]
> Why not adding the arming/disarming reason to vehicle_status? That would make it perfectly clear what the reason was.

I'm certainly not opposed to that. Explicit state transition reasons is something I'm hoping can be part of any state machine library/framework/etc adoption. https://github.com/PX4/Firmware/issues/10584

**Pull request merged**